doc/nrf/ug_matter.rst

```diff
@@ -19,7 +19,7 @@ Matter is in an early development stage and must be treated as an experimental f
 
 The |NCS| allows you to develop applications that are compatible with Matter.
 See :ref:`matter_samples` for the list of available samples or :ref:`Matter Weather Station <matter_weather_station_app>` for specific Matter application.
-If you are new to Matter, check also the tutorials on `DevZone`_.
+If you are new to Matter, you can also check the available video tutorials on `Nordic Semiconductor's YouTube channel`_, for example the one about `Developing Matter products with nRF Connect SDK`_.
```

Comment:
```suggestion
If you are new to Matter, you can follow along with the video tutorials on Nordic Semiconductor's YouTube channel, for example `Developing Matter products with nRF Connect SDK`_.
```

I would not use the first link to the main YouTube page. It has a lot of various content and doesn't direct the customers to Matter-specific videos, so it's not the ideal user experience. The second link is fine though. 

Is there actually more than one Matter-related video? If so, I wonder if we can talk to whoever is in charge of our YouTube channel and have them create a Matter video playlist and use that as a link here.

Reply:
@Avalei , please re-review.